support-frontend/assets/pages/[countryGroupId]/events/events.tsx

```diff
@@ -52,10 +55,14 @@ const tscs = css`
 `;
 
 export function Events() {
-	const searchParams = new URLSearchParams(window.location.search);
-	const eventId = searchParams.get('eventId') ?? '4180362';
-	const chk = searchParams.get('chk') ?? '9fa2';
+	const isTestUser = !!cookie.get('_test_username');
+	const shouldUseCode = isTestUser || !isProd();
+	const ticketTailorUrl = shouldUseCode
+		? 'https://www.tickettailor.com/events/guardianlivecode'
+		: 'https://tickets.theguardian.live/events/guardianlive';
 
+	const params = useParams();
+	const eventId = params.eventId;
```

Comment:
Is eventId  compulsary, ie must be a param in the url?

Reply:
the param is taken from the :eventId in the `routes` and `event/router`.

It is required as if we don't have it - what would we display?

Reply:
Maybe the Empty ticket tailor container on guardian scaffold?